tensorflow/lite/experimental/micro/kernels/depthwise_conv.cc

```diff
@@ -72,9 +72,9 @@ TfLiteStatus CalculateOpData(TfLiteContext* context, TfLiteNode* node,
     int exponent;
     QuantizeMultiplier(real_multiplier, &data->output_multiplier, &exponent);
     data->output_shift = -exponent;
-    CalculateActivationRangeUint8(params->activation, output,
-                                  &data->output_activation_min,
-                                  &data->output_activation_max);
+    (void)CalculateActivationRangeQuantized(context, params->activation, output,
```

Comment:
Instead of ignoring the errors in these calls, could you wrap this function in TF_LITE_ENSURE_STATUS

Here and in all the other files. Thanks!

Reply:
@suharshs: Your comment is handled in all the places, please check and approve, TIA!